server/API_STANDARD.md

```diff
@@ -2,7 +2,7 @@ OpenSupports can work completely fine without a frontend. You could only use thi
 This documentation is intended for application developers who are looking to integrate OpenSupports with other applications.
 
 ## Request/Response Interface
-The API server is located in the `api/` folder of your OpenSupports' instance folder.
+The API server is located in the `server/` folder of your OpenSupports' instance folder.
```

Comment:
Revert this. OpenSupports builded package uses `api/` and requests should be made to that place